triton-isa/src/parser.rs

```diff
@@ -1,12 +1,5 @@
 #![allow(clippy::needless_lifetimes)]
 #![allow(elided_named_lifetimes)]
```

Comment:
I added these attributes to suppress clippy warnings to fix this issues
![tritonvm-lifetimeissues](https://github.com/user-attachments/assets/cf18358d-fd9d-4fb8-aceb-c4f80f1784fc)